### PR TITLE
docs: use vite plugin 

### LINKED ISSUE #19: readme: add links to bundler plugins
vite &rarr; https://github.com/sodatea/vite-plugin-node-stdlib-browser

etc

related #14

### PR DESCRIPTION
**Pull request by milahu**
continue #17  

fix #19

todo: test "dont build in test" in CI. maybe add "npm run build && " before the test command

### CODE REVIEW

**Comment by happysalada** [comment]
@niksy what do you think about this PR ?

**Comment by niksy** [comment]
@happysalada I’ve left some comments for @milahu to review. When that’s done, I will merge PR.

**Comment by milahu** [comment]
where?

maybe you forgot to click "finish your review"

**Review by niksy — CHANGES_REQUESTED** [review]

File: test/index.js [hunk 1]
```diff
@@ -427,7 +427,6 @@ describe('Bundling', function () {
 	before(async function () {
 		if (shouldBundle) {
 			await execa('npm', ['install'], { cwd });
-			await execa('npm', ['run', 'build'], { cwd });
```
[the remark this answers is not in the document]
Comment (milahu) [reply test/index.js:430]: this is better for my workflow:

- build
- test
- modify the JS files produced by build
- test
- modify
- test

which is faster than running build on every test

> todo: test "dont build in test" in CI. maybe add "npm run build && " before the test command

File: example/esm/vite-plugin/vite.config.mjs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import nodePolyfills from 'vite-plugin-node-stdlib-browser';
+
+export default {
+	plugins: [
+		nodePolyfills(),
+	],
+}
+
+/* defineConfig: what is the benefit?
```
[the remark this answers is not in the document]
Comment (milahu) [reply example/esm/vite-plugin/vite.config.mjs:9]: yes. defineConfig has the advantage of intellisense for vite config

**Comment by milahu** [comment]
abandoned (sorry)

File: test/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (niksy) [reply test/index.js:430]: Don’t know what you mean by "your workflow", but this is the flow where you want to have proper test output. If you forget to run `npm run build` you will get wrong results.

**Comment by niksy** [comment]
Added reference to documentation.

**Pull request closed without merging**